ccd-definition/CaseField/CaseFieldLRspec.json

```diff
@@ -762,5 +762,27 @@
     "HintText": "For example 9 December 2020",
     "FieldType": "RespondToClaimAdmitPartLRspec",
     "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "specDefenceRoutePayFeeIfChangeDetails",
```

Comment:
Are you using this casefield anywhere? This appears to have been stopped use in another PR